pkg/controller/elasticsearch/validation/webhook.go

```diff
@@ -29,11 +30,13 @@ const (
 
 var eslog = ulog.Log.WithName("es-validation")
 
-func RegisterWebhook(mgr ctrl.Manager, validateStorageClass bool, exposedNodeLabels NodeLabels) {
+// RegisterWebhook will register the elasticsearch validating webhook.
+func RegisterWebhook(mgr ctrl.Manager, validateStorageClass bool, exposedNodeLabels NodeLabels, manaedNamespaces []string) {
```

Comment:
```suggestion
func RegisterWebhook(mgr ctrl.Manager, validateStorageClass bool, exposedNodeLabels NodeLabels, managedNamespaces []string) {
```
and where used further down ofc